projects/rawkode.academy/website/src/pages/watch/index.astro

```diff
@@ -24,9 +24,9 @@ Astro.response.headers.set(
 // Add build timestamp for debugging
 Astro.response.headers.set("X-Build-Time", new Date().toISOString());
 
-const latestVideosQuery = `
-  query GetLatestVideos($limit: Int!) {
-    getLatestVideos(limit: $limit) {
+const allVideosQuery = `
+  query GetAllVideos {
```

Comment:
Watch page should only show the latest 16, with a button to then work through the pages. 

I guess /watch/latest becomes redundant. Let's just have /watch and /watch?page=N